.pre-commit-config.yaml

```diff
@@ -15,6 +15,14 @@ repos:
     hooks:
       - id: isort
 
+  # Can run individually with `flynt [file]` or `flynt [source]`
+  - repo: https://github.com/ikamensh/flynt
+    rev: '0.78'
+    hooks:
+      - id: flynt
+        args: ["--fail-on-change", "--verbose"]
```

Comment:
As we discussed, this is a great starting point.  We'll play with these flags as we get more experience.